Formula/haproxy.rb

```diff
@@ -24,6 +27,13 @@ def install
       USE_ZLIB=1
       ADDLIB=-lcrypto
     ]
+    if build.with? "lua@5.3"
+      args << "USE_LUA=1"
+      args << "LUA_LIB_NAME=lua.5.3"
+      args << "LUA_LIB=#{Formula["lua@5.3"].lib}"
```

Comment:
`opt_lib`, `opt_include`, and so on. Surprised these are necessary once you tell it what version of `lua` you want it to use though, most build scripts are pretty smart in terms of handling the potential for multiple `lua`s sitting around.